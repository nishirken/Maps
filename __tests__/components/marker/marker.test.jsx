import { Marker } from 'Components';

describe('Rendered', () => {
    const TestMarker = shallow(<Marker />);

    it('Rendered', () => {
        expect(TestMarker).toMatchSnapshot();
    });
});
