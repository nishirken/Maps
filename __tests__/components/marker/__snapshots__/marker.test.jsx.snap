// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendered Rendered 1`] = `
<styled.svg
  center={false}
  viewBox="0 0 32 32"
>
  <path
    d="M17 0l-3 3 3 3-7 8h-7l5.5 5.5-8.5 11.269v1.231h1.231l11.269-8.5 5.5 5.5v-7l8-7 3 3 3-3-15-15zM14 17l-2-2 7-7 2 2-7 7z"
  />
</styled.svg>
`;
